tests/e2e/pv_services/test_del_mon_service_and_create_pvc.py

```diff
@@ -0,0 +1,192 @@
+import json
+import logging
+import pytest
+
+from ocs_ci.framework.testlib import (
+    skipif_ocs_version,
+    E2ETest,
+    tier4a,
+    ignore_leftovers,
+)
+from ocs_ci.helpers.disruption_helpers import Disruptions
+from ocs_ci.ocs import constants
+from ocs_ci.ocs.ocp import OCP, get_services_by_label
+from ocs_ci.ocs.resources.pod import get_mon_pods
+
+log = logging.getLogger(__name__)
+
+
+@tier4a
+@ignore_leftovers
+@skipif_ocs_version("<4.6")
+@pytest.mark.parametrize(
+    argnames=["interface"],
+    argvalues=[
+        pytest.param(
+            constants.CEPHBLOCKPOOL, marks=pytest.mark.polarion_id("OCS-2495")
+        ),
+        pytest.param(
+            constants.CEPHFILESYSTEM, marks=pytest.mark.polarion_id("OCS-2494")
+        ),
+    ],
+)
+class TestPvcCreationAfterDelMonService(E2ETest):
+    """
+    Tests to verify PVC creation after deleting
+    mon services manually
```

Comment:
consider running some background IOs during the test and verify that it's not stopped or hung after each mon changes its endpoints.